File: app/src/main/java/com/splatool/ikastyle/Dao/MainCategoryDao.java
package com.splatool.ikastyle.Dao;

import androidx.room.Dao;
import androidx.room.Query;

import com.splatool.ikastyle.Entity.MainCategory;

import java.util.List;

@Dao
public interface MainCategoryDao {
    @Query("SELECT * FROM MAST_MAIN_CATEGORY WHERE language_code = :languageCode ORDER BY id")
    List<MainCategory> getMainCategoryList(int languageCode);
}

File: app/src/main/java/com/splatool/ikastyle/dao/MainCategoryDao.kt
package com.splatool.ikastyle.dao

import com.splatool.ikastyle.entity.MainCategory
import androidx.room.*

@Dao
interface MainCategoryDao {
    @Query("SELECT * FROM MAST_MAIN_CATEGORY WHERE language_code = :languageCode ORDER BY id")
    open fun getMainCategoryList(languageCode: Int): List<MainCategory>
}